docs/configuration.rst

```diff
@@ -231,6 +231,16 @@ are only sent to remote storage once a task completes (including failure). In ot
 words, remote logs for running tasks are unavailable. Logs are stored in the log
 folder as ``{dag_id}/{task_id}/{execution_date}/{try_number}.log``.
 
+It's worth keeping in mind that since Airflow are using a ISO-8601 timestamps in the 
```

Comment:
Since Airflow uses the ISO-8601 timestamp format in the log path (i.e. `execution_date`), the default config is incompatible with Windows filesystems. In cases such as this, where the default logging configuration is not satisfactory, the user can override the default logging config class in ``airflow.cfg``